client/blocks/login/social.jsx

```diff
@@ -95,10 +88,14 @@ class SocialLoginForm extends Component {
 						onClick={ this.trackGoogleLogin } />
 				</div>
 
-				{ this.state.bearerToken && (
+				{ this.props.isSocialAccountCreating &&
+					<InfoNotice text={ this.props.translate( 'Creating your account…' ) } />
```

Comment:
If we really want to proceed with this approach, I would prefer to put that component in [client/components/global-notices](https://github.com/Automattic/wp-calypso/tree/master/client/components/global-notices) folder.